jupyterlab_gcedetails/src/components/auth_token_retrieval.tsx

```diff
@@ -23,5 +25,12 @@ export function authTokenRetrieval(): Promise<string> {
       '_authPopup',
       'left=100,top=100,width=400,height=400'
     );
+    timer = setInterval(function() {
```

Comment:
So apparently if I try setting a function for authPopup.beforeUnload, it won't work because the child window's domain is different from the parents. I searched it up and people are saying this is the way to go. Tested it and it works.

Reply:
Great investigation. I'm good with it!